public/app/features/explore/ExploreDrawer.tsx

```diff
@@ -35,15 +35,14 @@ const getStyles = stylesFactory((theme: GrafanaTheme2) => {
       animation: 0.5s ease-out ${drawerSlide(theme)};
     `,
     rzHandle: css`
-      background: ${theme.colors.secondary.main};
```

Comment:
Is this removed on purpose?

Reply:
In UI, the handle is not visible now. 
Now:
<img width="929" alt="image" src="https://user-images.githubusercontent.com/30407135/228503549-7b5222a2-fb7f-4305-97f9-05cbef77bf2e.png">

Before: 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/30407135/228503614-bc2567d9-2c6f-4242-8ea1-1ab9c242fad8.png">

Reply:
No, I removed it by mistake. Now it looks fine again.